build-farm/platform-specific-configurations/aix.sh

```diff
@@ -68,7 +68,22 @@ then
     fi
   fi
   export JDK_BOOT_DIR=$JDK10_BOOT_DIR
+fi
 
+if [ "${JAVA_TO_BUILD}" == "${JDK12_VERSION}" ] || [ "${JAVA_TO_BUILD}" == "${JDKHEAD_VERSION}" ];
+then
+  if [ ! -d "$JDK11_BOOT_DIR" ]; then
+    export JDK11_BOOT_DIR="$PWD/jdk-11"
+    if [ ! -d "$JDK11_BOOT_DIR/bin" ]; then
+      mkdir -p "$JDK11_BOOT_DIR"
+      wget -q -O - "https://api.adoptopenjdk.net/v2/binary/releases/openjdk11?os=aix&release=latest&arch=${ARCHITECTURE}&heap_size=normal&openjdk_impl=openj9" | tar xpzf - --strip-components=2 -C "$JDK11_BOOT_DIR"
```

Comment:
should this be hotspot by default?

Reply:
Maybe, but https://api.adoptopenjdk.net/v2/binary/releases/openjdk11?os=aix&openjdk_impl=hotspot returns no builds.

Reply:
OK, we'll look at getting a Hotspot build out but lets not block this